### PR TITLE
#1204 Add path to stack exchange and slack

### PR DESCRIPTION
**Pull request by whatnick**
### Reason for this pull request
Partial work towards #1204 

### Proposed changes
Doc lines in issue template.

### CODE REVIEW

**Review by woodcockr — APPROVED** [review]
LGTM

**Review by robbibt — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -17,4 +17,6 @@
 
 
 > **Note:** Stale issues will be automatically closed after a period of six months with no activity. 
-> To ensure critical issues are not closed, tag them with the Github `pinned` tag .
+> To ensure critical issues are not closed, tag them with the Github `pinned` tag.
+> If you are a community member and not a maintainer please escalate this issue to maintainers via
+> [GIS StackExchange](https://gis.stackexchange.com/questions/tagged/open-data-cube) or [Slack](https://join.slack.com/t/opendatacube/shared_invite/zt-d6hu7l35-CGDhSxiSmTwacKNuXWFUkg).
```
Comment (alexgleith) [inline .github/ISSUE_TEMPLATE.md]: The slack link is not https... I should probably put cloudfront on it with a cert!

Comment (alexgleith) [reply .github/ISSUE_TEMPLATE.md]: Oh, the slack link is http://slack.opendatacube.org too

**Review by alexgleith — CHANGES_REQUESTED** [review]
Slack URL should be different.

**Comment by whatnick** [comment]
The Slack inviter is pretty gatekeepery since it works only with GA and
CSIRO emails and fails silently with others. The link in this PR is the
Slack native inviter and more widely usable.

**Comment by benjimin** [comment]
_"If you are a community member and not a maintainer please escalate this issue to maintainers via GIS StackExchange or Slack"_

What about wording it as: "_This issue tracker is for fixes and improvements to the ODC core library. For support using ODC, consider asking a question on GIS StackExchange or the ODC Slack._"?

My thoughts:
* ODC questions on GIS stack exchange are very likely to be **closed and downvoted**, like [this one](https://gis.stackexchange.com/q/413614/59848) was yesterday.
* Github _should_ be the correct place for escalating something to the attention of the maintainers.
* The community is performing a service by reporting issues. If we communicate expectations that are unnecessarily onerous (or present an exclusionary vibe), issues regarding the project will go unreported.

**Comment by alexgleith** [comment]
Nah, this works for everyone @whatnick: http://slack.opendatacube.org/

**Comment by alexgleith** [comment]
Yo @whatnick can you please update the Slack joining link to `http://slack.opendatacube.org`. Then we can merge this!

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (alexgleith) [inline .github/ISSUE_TEMPLATE.md]:
```suggestion
> [GIS StackExchange](https://gis.stackexchange.com/questions/tagged/open-data-cube) or [Slack](http://slack.opendatacube.org).
```

**Review by alexgleith — APPROVED** [review]

**Pull request merged**